dev-docs/bidders/connatix.md

```diff
@@ -32,7 +32,38 @@ sidebarType: 1
 | `placementId` | required | Placement id | `'ed8a9c16-88ea-4724-aa0d-013c3e595e49'` | `string` |
 | `bidfloor` | optional | Floor price | `2.5` | `float` |
 
-#### Example
+#### Example Video Media Type
+
+```js
+var adUnits = [
+  {
+    code: "1",
+    mediaTypes: {
+      video: {
+        context: "instream",

```

Comment:
please add the width and height params (or sizes) which is recommended.

I would add here in the example all the supported recommended params